src/include/timestamp_inline.h

```diff
@@ -65,6 +65,22 @@
         (tw)->stop_txn = (upd)->txnid;                           \
     } while (0)
 
+/* Copy the start values of a time window from another time window. */
+#define WT_TIME_WINDOW_COPY_START(tw1, tw2)                \
```

Comment:
I prefer `source` and `dest` rather than `tw1` and `tw2` to be clear which direction the initialization happens in. I also wonder if there is a reason to use a macro rather than an inline function? I guess we'd need two (or four) inline functions, to deal with different data structures with time windows - though I feel like that type checking makes the duplication worthwhile.

Reply:
The update and time window they have different fields in the data structure so we would still get a compile failure when we pass the wrong type. I use macro just to be consistent to what we already have.